dapr_agents/workflow/agentic.py

```diff
@@ -109,6 +110,7 @@ def model_post_init(self, __context: Any) -> None:
         self._text_formatter = ColorTextFormatter()
 
         # Initialize state store client (used for persisting workflow state to Dapr)
+        # Why make a state store client and dapr_client here?
```

Comment:
mmm I guess here the idea was not to open and close connections for agent state and have a persistent one. However, I believe that with that we have to handle the "closing" of the connection the right way vs using `with daprclient` do X. right?